tests/scripts/test_coverage_pipeline_integration.py

```
@@ -165,6 +166,89 @@ def test_pipeline_handles_missing_coverage_xml() -> None:
+        (workdir / "test-summary.json").write_text(json.dumps(summary))
+
+        env = {
+            **os.environ,
+            "COVERAGE_PERCENTAGE": "0.0",
+            "GITHUB_EVENT_NAME": "pull_request",
+            "GITHUB_REPOSITORY": "lgtm-hq/py-lintro",
+            "GITHUB_RUN_ID": "123456",
+            "GITHUB_SHA": "abc123",
+            "JOB_RESULT": "failure",
+        }
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)